upgrading/upgrade-operator.adoc

```diff
@@ -9,15 +9,11 @@ toc::[]
 [role="_abstract"]
 Upgrades through the {rh-rhacs-first} Operator are performed automatically or manually, depending on the *Update approval* option you chose at installation.
 
-{product-title-short} 4.0 includes a significant architectural change, moving Central’s database to PostgreSQL. Because of this change, {product-title-short} 4.0 Operator is published by a new subscription channel. Therefore, as part of the upgrade instructions, you must manually change the subscription channel to upgrade from {product-title-short} 3.74 to {product-title-short} 4.0.
+To get upgrade instructions, see the documentation for your product version. Follow these guidelines when upgrading:
 
-//If you installed {product-title-short} using the Operator and selected *Automatic* in the *Update approval* field, {product-title-short} is automatically updated when a new software version is released. If you selected *Manual*, you must approve subsequent Operator updates by using Operator Lifecycle Manager (OLM). For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/administrator-tasks#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending Operator update].
-
-[IMPORTANT]
-====
-* Because of the database related changes introduced in {product-title-short} 4.0, even if you have selected *Automatic* in the *Update approval* field, you must manually upgrade to {product-title-short} 4.0.
-* You must be using {product-title-short} 3.74 to upgrade to {product-title-short} 4.0. If you are using a version older than 3.74, you must first upgrade to {product-title-short} 3.74 and then upgrade to {product-title-short} 4.0.
-====
+* If Central is earlier than 3.74, you must upgrade to 3.74 before upgrading to a 4.x release. For upgrading Central from version 3.74 to 4.x, see the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.74/html/upgrading/index[upgrade documentation for version 3.74].
+* For Operator- and Helm-based Central deployments, you can upgrade from release 3.74 to any latest patch of {product-title-short} version 4.0 through 4.4. However, for full functionality, upgrade to the latest version.
```

Comment:
Same suggestion as I did for Helm.

Reply:
An alternative wording.  Basically the same

* When upgrading Helm-based Central deployments from release 3.74, first ensure Operator upgrade mode is set to Manual, upgrade the Operator to 4.0 following the procedure in the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/4.0/html/upgrading/upgrade-operator[upgrade documentation for version 4.0], and ensure Central comes up online. Once the upgrade to 4.0 is complete, Red Hat recommends upgrading central to the latest version for full functionality.